components/transfer/demo/custom-select-all-labels.md

```diff
@@ -0,0 +1,68 @@
+---
+order: 7
+title:
+  zh-CN: 自定义全选文字
+  en-US: Custom Select All Labels
+---
+
+## zh-CN
+
+自定义穿梭框全选按钮的文字。
+
+## en-US
+
+Custom the labels for select all checkboxs.
+
+```jsx
+import { Transfer } from 'antd';
+
+class App extends React.Component {
+  state = {
+    mockData: [],
+    targetKeys: [],
+  };
+
+  componentDidMount() {
+    this.getMock();
+  }
+
+  getMock = () => {
```

Comment:
No need to create data in did mount. We should simplify this demo since it's for debug usage.

Reply:
OK, will update it. 
Is it needed to add `debug` property in .md demo?